client/dev-tools-promo/components/dev-tools-promo.tsx

```diff
@@ -0,0 +1,96 @@
+import { Card } from '@automattic/components';
+import { localizeUrl } from '@automattic/i18n-utils';
+import { Button } from '@wordpress/components';
+import { translate } from 'i18n-calypso';
+import CardHeading from 'calypso/components/card-heading';
+import { useSelector } from 'calypso/state';
+import { getSiteSlug } from 'calypso/state/sites/selectors';
+import { getSelectedSiteId } from 'calypso/state/ui/selectors';
+import './style.scss';
+
+type PromoCardProps = {
+	title: string;
+	text: string;
+	href: string;
+};
+
+const DevToolsPromo = () => {
+	const siteId = useSelector( getSelectedSiteId );
+	const siteSlug = useSelector( ( state ) => getSiteSlug( state, siteId ) ) || '';
+
+	const PromoCard = ( { title, text, href }: PromoCardProps ) => (
+		<Card className="dev-tools-promo__card">
+			<CardHeading>{ title }</CardHeading>
+			<p>{ text }</p>
+			<a href={ href }>{ translate( 'Learn more' ) }</a>
```

Comment:
How about to open the doc by the modal? We can iterate it later 🙂

Reply:
hmm you mean it would open in the help center modal and remain in calypso? What do you think @lucasmendes-design

Reply:
YES! People may just want to know more about these features but it seems not friendly if they have to go back and forth. It's not a blocker

Reply:
+1 to opening docs links in the help center modal.